ramls/event.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Event data model",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "type": "string"
+    },
+    "eventType": {
+      "description": "Unique Event type, human-readable String, not UUID",
+      "type": "string"
+    },
+    "metadata": {
+      "description": "Event metadata",
+      "type": "object",
+      "properties": {
+        "tenantId" : {
+          "description": "Tenant id",
+          "type": "string"
+        },
+        "eventTTL": {
+          "description": "Default time-to-live (TTL) for event in minutes",
```

Comment:
Remove the "Default" please since this is a particular TTL for this event.

Reply:
Done, thanks